projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.scss

```diff
@@ -1,20 +1,87 @@
 :host {
   display: flex;
-  align-items: center;
-  flex-direction: row;
+  flex-direction: column;
   flex-wrap: wrap;
 
+  .dropdown-label {
+    display: none;
+    align-items: center;
+  }
+
+  hr {
+    display: none;
+  }
+
   .view-slider {
     margin-left: -1rem;
     margin-bottom: 1.5rem;
+    white-space: nowrap;
   }
 
   .input-group {
     margin-right: 1.5rem;
-
+    
     .nav-input {
       margin-right: 1.5rem;
       margin-bottom: 1.5rem;
     }
   }
+  
+  &.dropdown {
+    .dropdown-label {
+      display: flex;
+      height: 3rem;
+      cursor: pointer;
+    }
+    
+    .stage-nav {
+      display: flex;
+      flex-direction: column;
+      border-radius: 0.25rem;
+      width: 16.5rem;
+      position: absolute;
+      top: 3rem;
+  
+      :first-child {
+        margin-top: 0.5rem;
+      }
+  
+      :last-child {
+        margin-bottom: 0.5rem;
+      }
+      
+      &.hidden {
+        display: none;
+      }
+      
+      hr {
+        display: inherit;
+        width: 100%;
+        margin: 0;
+        border-top: 3px solid;
+      }
+    
+      .input-group {
+        margin-left: 1.5rem;
+        margin-right: 0;
+        display: flex;
```

Comment:
Might look better to have one option per row. I.e. `flex-direction: column`

Reply:
Leonard wanted two options per row, so going to leave it this way for now.